depends/packages/native_libdmg-hfsplus.mk

```diff
@@ -1,8 +1,8 @@
 package=native_libdmg-hfsplus
-$(package)_version=0.1
-$(package)_download_path=https://github.com/theuni/libdmg-hfsplus/archive
-$(package)_file_name=libdmg-hfsplus-v$($(package)_version).tar.gz
-$(package)_sha256_hash=6569a02eb31c2827080d7d59001869ea14484c281efab0ae7f2b86af5c3120b3
+$(package)_version=3e5fd3fb56bc9ff03beb535979e33dcf83fe1f70
+$(package)_download_path=https://github.com/fanquake/libdmg-hfsplus/archive
```

Comment:
Any reason why we're using your fork instead of upstream?

Reply:
I'm still applying one commit on top of upstream: https://github.com/fanquake/libdmg-hfsplus/commit/3e5fd3fb56bc9ff03beb535979e33dcf83fe1f70

Although now that there's only one commit left we could put the patch in depends and point back to upstream if you'd prefer?